apps/web/src/views/Farms/components/FarmTable/FarmTable.tsx

```diff
@@ -71,15 +71,13 @@ const TableContainer = styled.div`
 `
 
 const getV2FarmEarnings = (farm: V2Farm) => {
-  let earnings
   const existingEarnings = new BigNumber(farm.userData.earnings)
+  let earnings: BigNumber = existingEarnings
 
   if (farm.boosted) {
     const proxyEarnings = new BigNumber(farm.userData?.proxy?.earnings)
 
     earnings = proxyEarnings.gt(0) ? proxyEarnings : existingEarnings
-  } else {
-    earnings = existingEarnings
   }
 
```

Comment:
only changed this becase i was getting a non null check flag when trying to commit